xfs/parse_test.go

```diff
@@ -11,23 +11,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package xfs_test
```

Comment:
A nit as the author of this package: I would prefer this to remain `xfs_test` so we are forced to exercise the exported API. It doesn't appear any unexported calls are used anyway.

Reply:
I agree this makes sense since we're trying to validate API usage.  I have changed parse_test and xfs_test back to using the xfs_test package.  I'll review some of the other packages to see where it makes sense to follow the same pattern.